lib/password.flow

```diff
@@ -5,6 +5,17 @@ export {
 		password : string, 
 		hash : string 
 	) -> bool = FlowPassword.validateHash;
+
+	// < 0: means no expiration
+	// 0: use default expiration time (84600 sec -> 1 day)
+	// > 0: use specified value 
+	native getPasswordValidationStringExp : (exp: int) -> string = FlowPassword.getPasswordValidationString;
```

Comment:
Instead of "exp" it is better to call it "expirationS". (I assume it is in seconds).